components/PomodoroTimer.tsx

```diff
@@ -188,10 +201,16 @@ export function PomodoroTimer() {
                     rounded="circle"
                     shadow="md"
                     borderless
-                    onPress={() => startTimer()}
+                    onPress={() => {
+                        if (playing)
+                            return pauseTimer();
+
+                        return startTimer();
+                    }}
                 >
-                    <Icon name="play" color="white" fontFamily="Feather" fontSize="2xl" />
+                    {playingStateIcon}
```

Comment:
Follow the previous suggestion, adding the ternary inside the `name` is more clean and more readable

```suggestion
                    <Icon name={playing ? "pause" : "play"} color="white" fontFamily="Feather" fontSize="2xl" />
```